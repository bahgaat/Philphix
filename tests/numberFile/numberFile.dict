23456789098765321123456789876543            345678998765432345678876543
34567898765                                     333333333333333333333333333333333
1234567098765432112345                                      333333333333333333333333333333333
303902900000000000000000000000000333333333333333333333333333                   22222222222
333333333333                         5
3 4
7 8
777  66
666  44
5 6
9 6
2345654        345678998765432345678876543
345654      34543455
45678765            4567654
4566543                      4444444
4446765                          333333333333333333333333333333332222222222
098765       5555
456543                  44444444444444
5456           09090
2343212             555555
4543         2343212
22       44
876          5555
66           66666
523432                222222
