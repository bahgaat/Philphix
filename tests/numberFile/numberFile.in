23456789098765321123456789876543   34567898765
333333333333
4543
