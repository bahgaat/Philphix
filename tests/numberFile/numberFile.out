345678998765432345678876543   333333333333333333333333333333333
5
2343212
